test/parallel/test-timers-unref-call.js

```diff
@@ -15,8 +15,6 @@
 var common = require('../common');
 
 var Timer = process.binding('timer_wrap').Timer;
-Timer.now = function() { return ++Timer.now.ticks; };
```

Comment:
Unfortunately, I don't fully understand what Bob meant. However, the test runs without it, i.e. check it out correctly `this._onTimeout`.